lib/mini_profiler_rails/railtie.rb

```diff
@@ -24,7 +24,12 @@ def self.initialize!(app)
 
     c.skip_paths ||= []
 
-    c.skip_paths << app.config.assets.prefix if serves_static_assets?(app)
+    begin
+      c.skip_paths << app.config.assets.prefix if serves_static_assets?(app)
+    rescue
+      # app.config did not repond_to? :assets
```

Comment:
> first line is !app.respond_to(:assets)

I noticed this as well, and was somewhat confused by it. Notice here we are checking if app.*config* responds to :assets, not just `app`. What is the reason for this?